content/projekte/_index.md

```diff
@@ -9,16 +9,80 @@ outputs:
+Aus gesammelten Informationen und visualisierten Karten
+in [Leipzig](https://opengovtech.de/cfg-ziele/projekte/le-kitas_und_schulen_in_leipzig/), [Köln](https://opengovtech.de/cfg-ziele/projekte/cologne-schulen-in-koeln/),  [Berlin](https://opengovtech.de/cfg-ziele/projekte/be-berliner-schulen/),
+im [Ruhrgebiet](https://opengovtech.de/cfg-ziele/projekte/schulradar-ruhrgebiet/) und [Nordrhein-Westfahlen](https://opengovtech.de/cfg-ziele/projekte/schulen-in-nrw/),
+haben wir [JedeSchule.de](https://jedeschule.de/) gebaut. Hier sind
+Informationen zu 30.000 Schulen vereint.
+
+### Wo ist der nächste Wochenmarkt?  
+
+Hier findest du die Antwort
+für[Leipzig](https://opengovtech.de/cfg-ziele/projekte/le-woistmarkt/),  [Karlsruhe](https://opengovtech.de/cfg-ziele/projekte/ka-woistmarkt/)  und  [Berlin](https://opengovtech.de/cfg-ziele/projekte/be-woistmarkt/).
+So eine „Wo ist..."-Karte funktionierte in den ersten Pandemiejahren
+auch super mit  Corona-Testzentren  und ist ein gutes Beispiel für
+die Wiederverwendbarkeit von Open-Source-Software in  Berlin,
+Düsseldorf, Halle, Kelletal, Leipzig, Lemgo
+und  [Hessen](https://opengovtech.de/cfg-ziele/projekte/wo-ist-testzentrum/).
+
+### Welche Qualität hat mein Trinkwasser?  
+
+Falls du
+in  [Berlin](https://opengovtech.de/cfg-ziele/projekte/be-trinkwasser/),  [Köln](https://opengovtech.de/cfg-ziele/projekte/trinkwasser/),  [Leipzig](https://opengovtech.de/cfg-ziele/projekte/le-trinkwasser/),  [Potsdam](https://opengovtech.de/cfg-ziele/projekte/p-trinkwasser/)  oder  [Heilbronn](https://opengovtech.de/cfg-ziele/projekte/hn-trinkwasser/)  wohnst,
+können wir dir diese Frage sofort beantworten.
+
+### Wo ist in Köln der nächste Defibrillator?  
+
+[Hier](https://opengovtech.de/cfg-ziele/projekte/muc-defis/).
+
+### Wie viele Bäume gibt's in meiner Stadt?  
+
+In  [Hamburg](https://opengovtech.de/cfg-ziele/projekte/hh-baumkataster/),  [Magdeburg](https://opengovtech.de/cfg-ziele/projekte/magdeburg-baumfreunde-md/),  [Karlsruhe](https://opengovtech.de/cfg-ziele/projekte/baumkataster/),  [Köln](https://opengovtech.de/cfg-ziele/projekte/cologne-baumkataster/)  haben
+wir das Baumkataster visualisiert. Du kannst kollaborativ daran
+mitarbeiten.  
+
+### Wie kann ich Bäume vor dem Vertrocknen retten?  
+
+Wenn du deinen Kiez grün halten möchtest, dann schau doch mal bei
+den Gieß-Apps
+von [Köln](https://opengovtech.de/cfg-ziele/projekte/cologne-giesst/),  [Leipzig](https://opengovtech.de/cfg-ziele/projekte/le-giessdeinviertel/),  [Berlin](https://opengovtech.de/cfg-ziele/projekte/grossstadtbaum/)  und  [Heilbronn](https://opengovtech.de/cfg-ziele/projekte/hn-kastanienapp/)  vorbei.  
+
+### Wann fährt der günstigste Zug?  
+
+Mit [bahn.guru](https://bahn.guru/) oder der [Bahn- und
+Buspreiskarte](https://codefor.de/projekte/pricemap-eu/) kannst du
+das schnell herausfinden. Auch
+alle [Direktverbindungen](https://codefor.de/projekte/zug-direktverbindungen/)  aus
+deiner Stadt lassen sich visualisieren.
+
```

Comment:
```suggestion
{{</ paragraph-center >}}
```

Reply:
Hier bin ich mir nicht sicher, wann ich den paragraph-center auf und zumachen muss. Weil ich vermute, dass er vor einer h1 zu, und nach einer h1 wieder aufgemacht werden muss, habe ich ihn hier zugemacht und in Zeile 87 wieder aufgemacht.

Reply:
Muss man bei Hugo tatsächlich nicht. Wir können shortcodes auch in anderen shortcodes benutzen.